svc-aggregation/rpc/common.go

```diff
@@ -55,7 +55,7 @@ func GetAggregator() *Aggregator {
 			EventNotification:       agmessagebus.Publish,
 			GetAllKeysFromTable:     agmodel.GetAllKeysFromTable,
 			GetConnectionMethod:     agmodel.GetConnectionMethod,
-			GetPluginData:		 agmodel.GetPluginData,
+			GetPluginMgrAddr:	 agmodel.GetPluginMgrAddr,
```

Comment:
this actual function agmodel why this name is changed

Reply:
I earlier committed changes to use mockup function for GetPluginData in UT. Now replaced that call with call to new function GetPluginMgrAddr()

Reply:
I think the question was why this change

Reply:
GetPluginData is an existing function widely used and mocking it is interfering with existing UTs. So created GetPluginMgrAddr which is mocked in UT (specifically for this fix) and just wraps GetPluginData in normal execution.